src/libexpr/eval.cc

```diff
@@ -273,7 +273,7 @@ EvalState::EvalState(
 
             /* Apply access control if needed. */
             if (settings.restrictEval || settings.pureEval)
-                accessor = AllowListSourceAccessor::create(accessor, {},
+                accessor = AllowListSourceAccessor::create(accessor, {}, {},
```

Comment:
Is it possible to add a test case for empty repositories?

Reply:
Added a test.